Classes/TWMessageBarManager.m

```diff
@@ -266,6 +266,9 @@ - (void)showMessageWithTitle:(NSString *)title description:(NSString *)descripti
     {
         [self showNextMessage];
     }
+    
+    // Focus on this view when in voiceover
+    UIAccessibilityPostNotification(UIAccessibilityLayoutChangedNotification, messageView);
```

Comment:
Are you sure it's the right behavior to focus the view here. I feel that since these aren't modal alerts they shouldn't steal the focus from the user. 

Would it make more sense to have something like 

```
    UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, @"This is an alert")
```

Apple suggests using this notification type like so:
> Use this notification to provide accessibility information about events that do not update the application user interface (UI), or that update the UI only briefly.
That only briefly part sounds appropriate

Reply:
So I tried that first and the issue is that on screens like the job application screen, because we are dismissing a modal screen and showing a message the announcement just gets interrupted by the focus jumping to something(usually the back button?) on the screen behind it.

Reply:
Oh wow, okay, I hadn't considered that context. Good thinking. That's got to be a fairly large (or at least important) subsection of the times this would get used.  This definitely seems like the safer option then